dali/operators/reader/loader/loader.h

```diff
@@ -109,9 +110,11 @@ class Loader {
   }
 
   template <typename T>
-  std::enable_if_t<!std::is_same<T, Tensor<CPUBackend>>::value>
+    std::enable_if_t<!(std::is_same<T, Tensor<CPUBackend>>::value ||
+                       std::is_same<T, Tensor<GPUBackend>>::value)>
   PrepareEmptyTensor(T&) {
-    constexpr bool T_is_Tensor = std::is_same<T, Tensor<CPUBackend>>::value;
+    constexpr bool T_is_Tensor = (std::is_same<T, Tensor<CPUBackend>>::value ||
```

Comment:
this is always false, isn't it?

Reply:
I guess that T may not necessarily need to be `Tensor` (however now it is). Should I change it?